src/AvroHelper.ts

```diff
@@ -83,10 +83,10 @@ export default class AvroHelper implements SchemaHelper {
   }
 
   updateOptionsFromSchemaReferences(
-    options: ProtocolOptions,
-    referredSchemas: AvroConfluentSchema[],
+    referencedSchemas: AvroConfluentSchema[],
+    options?: ProtocolOptions,
   ): ProtocolOptions {
     const opts = options ?? {}
-    return { ...opts, [SchemaType.AVRO]: { ...opts[SchemaType.AVRO], referredSchemas } }
+    return { ...opts, [SchemaType.AVRO]: { ...opts[SchemaType.AVRO], referencedSchemas } }
```

Comment:
Why not default `options = {}` to empty object instead of line 89? That way it also becomes optional.